tests/build/build_above_src.srctree

```diff
@@ -0,0 +1,40 @@
+# Tests the out of place (ie with build_dir set) build when a src file is
+# "above" the setup.py (ie it's path begins with dots). 
+# For example, "../../../src/foo.pyx". See #2764 for details.
+
+cd a/b/c/d; PYTHON setup.py build_ext -b ../../pkg -t .
```

Comment:
The directory names here do not work on Windows.

Reply:
I hardly ever use windows, so I'm not quite sure what you're getting at, nor how to fix it. Can you please give me a hint/pointer on how to make this line platform independent?

Reply:
Yeah, according to Appveyor, `cd a/b/c/d` does not work but needs to use `\`. Separate `cd a`, `cd b`, … commands would be an easy way to fix it, I guess.